packages/web-app-files/src/components/FilesList/QuickActions.vue

```diff
@@ -39,7 +40,9 @@ export default defineComponent({
 
     const filteredActions = computed(() => {
       return unref(extensionRegistry)
-        .requestExtensions<ActionExtension>('action', { scopes: ['resource.quick-action'] })
+        .requestExtensions<ActionExtension>('action', {
+          extensionPointIds: [quickActionsExtensionPointId]
+        })
```

Comment:
Nice idea 💪 did that now in the latest commit. Needed to finally change the signature of the `requestExtensions` function for this, so that it only expects one `options` object (which can contain either an `extensionPoint` object or an `extensionType` string and optionally a list of `extensionPointIds`).